src/supremm/datasource/pcp/pcpsummarize.py

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/env python
+""" Summarize module for PCP datasource """
+
+import datetime
+
+from ctypes import c_uint
+from pcp import pmapi
+import cpmapi as c_pmapi
+import time
+import logging
+import traceback
+from supremm.plugin import NodeMetadata
+from supremm.rangechange import RangeChange, DataCache
+from supremm.summarize import Summarize
+from supremm.datasource.pcp.pcpcinterface import pcpcinterface
+
+import numpy
+import copy
+
+VERSION = "2.0.0"
```

Comment:
This version should only change if the output schema changes. Please don't change this!

Reply:
Resolved in:
https://github.com/ubccr/supremm/pull/259/commits/a34241f9050f4e6fcc5b9fb084199663ce3c9fe2
https://github.com/ubccr/supremm/pull/259/commits/e928967e850ddb22068e1e23ae33fa39e58e0b23
https://github.com/ubccr/supremm/pull/259/commits/c7b17840d6afddd9945104bdc48378a428543fb2